src/app/work-item/work-item-list-entry/work-item-list-entry.component.ts

```diff
@@ -51,9 +52,14 @@ export class WorkItemListEntryComponent implements OnInit {
   loggedIn: Boolean = false;
 
   constructor(private auth: AuthenticationService,
+              private broadcaster: Broadcaster,
               private router: Router,
               private workItemService: WorkItemService,
               private logger: Logger) {
+                 this.broadcaster.on<string>('logout')
```

Comment:
Should this really be done in the constructor. There are several different life cycle methods that you may want to consider, like ngOnInit. You really need to ask yourself when do you want this method called.